lib/ronin/web/cli/commands/diff.rb

```diff
@@ -109,25 +109,26 @@ def read(source)
 
           private
 
-          def load_doc(page1, page2)
-            case options[:format].to_sym
+          #
+          # Loads the given html or xml sources
+          #
+          # @param [String] page1
+          #   The URL or file path of the original page.
+          #
+          # @param [String] page2
+          #   The URL or file path of the modified page.
+          # @return [@Nokogiri::HTML, @Nokogiri::HTML] or [@Nokogiri::XML, @Nokogiri::XML]
+          #
+          def load_docs(page1, page2)
+            case options[:format]
             when :html
-              [load_html(page1), load_html(page2)]
+              [Nokogiri::HTML(read(page1)), Nokogiri::HTML(read(page2))]
```

Comment:
Since this is technically duplicate code, maybe `load_docs` could be refactored as a `load_doc(page)` method. Then at the top of `run`, you would have:

```ruby
doc1 = load_doc(page1)
doc2 = load_doc(page2)
```

Whichever you think is easier.